### PR TITLE
Add Another notes app and K-9 Mail icons

### PR DESCRIPTION
**Pull request by benteg**
## Description
Added icons for Another notes app and K-9 Mail.

## Type of change

:x: Bug fix (non-breaking change which fixes an issue)
:white_check_mark: : Icon addition (non-breaking change that adds/modifies Lawnicons's icons)
:x: General change (non-breaking change that doesn't fit the above categories like copyediting)

### Icons added
* Another notes app (`com.maltaisn.notes.sync`)
* K-9 Mail (``)

### Icons linked
* Another notes app (linked `com.maltaisn.notes.sync` to `@drawable/another_notes_app`)
* K-9 Mail (linked `com.fsck.k9` to `@drawable/k9_mail`)

### CODE REVIEW

File: app/src/main/res/xml/grayscale_icon_map.xml [hunk 1]
```diff
@@ -631,7 +634,7 @@
     <icon drawable="@drawable/openapk" package="com.dkanada.openapk" name="OperaAPK" />
     <icon drawable="@drawable/openfoodfacts" package="openfoodfacts.github.scrachx.openfood" name="OpenFoodFacts" />
     <icon drawable="@drawable/openkeychain" package="org.sufficientlysecure.keychain" name="OpenKeychain" />
-    <icon drawable="@drawable/opentracks" package="de.dennisguse.opentracks" name="OpenTracks" />
```
Comment (Chefski) [inline app/src/main/res/xml/grayscale_icon_map.xml:634]: Was this line deleted intentionally? Could you add it back please?

**Review by Chefski — CHANGES_REQUESTED** [review]
Hey! Thanks for contributing! Icons look great but I've got some small changes to ask.
Could you increase the content area size for the following icons:

- another_notes_app
- k9_mail
- openweather
- samsung_health

image

File: app/src/main/res/xml/grayscale_icon_map.xml (hunk 1 above)
Comment (benteg) [reply app/src/main/res/xml/grayscale_icon_map.xml:634]: Yeah, that was by mistake... I'll add it back.

Comment (benteg) [reply app/src/main/res/xml/grayscale_icon_map.xml:634]: To which size exactly?

**Review by Chefski — APPROVED** [review]
Thank you

**Pull request merged**